tests/unit_tests/commands/android/testIndex.js

```diff
@@ -440,6 +440,9 @@ describe('test getConfigFromOptions', function() {
 
     configs = androidSetup.getConfigFromOptions({mode: true, browsers: []});
     assert.deepStrictEqual(configs, {});
+
+    configs = androidSetup.getConfigFromOptions({standalone: true, browsers: false});
+    assert.deepStrictEqual(configs, {browsers: 'none'});
```

Comment:
Add tests for the following cases as well:
* If no `--browsers` flag is passed, then it should be set to `none`.
* If only chrome is passed, it should be set to 'chrome'.
* If both chrome and firefox are passed, it should be set to 'both'.
* If some random browser is passed (ex. brave), it should be set to 'none'.